src/test/java/org/apache/datasketches/kll/KllMiscItemsTest.java

```diff
@@ -205,13 +221,64 @@ public void viewCompactionAndSortedView() {
     GenericSortedViewIterator<String> itr = sv.iterator();
     println("### SORTED VIEW");
     printf("%12s%12s\n", "Value", "CumWeight");
+    long[] correct = {2,2,2,2,2,2,2,2,2,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1};
+    int i = 0;
     while (itr.next()) {
       String v = itr.getQuantile();
       long wt = itr.getWeight();
       printf("%12s%12d\n", v, wt);
+      assertEquals(wt, correct[i++]);
     }
   }
 
+  @Test //set static enablePrinting = true for visual checking
+  public void checkWeightedUpdates1() {
+    int k = 20;
+    int weight = 127;
+    String item = "10";
+    KllItemsSketch<String> sk = KllItemsSketch.newHeapInstance(20, Comparator.naturalOrder(), serDe);
```

Comment:
should be k, which is defined above and never used

Reply:
Fixed